docs/app/pages/components/components-sections/tables/filters/snippets/app.html

```diff
@@ -20,21 +20,27 @@
     <tbody>
         <tr class="clickable" *ngFor="let document of documents">
             <td>
-                <div class="listview-page-icon"></div>
+                <ux-icon
```

Comment:
I'm not sure about the document icon on each row in the example:

![image](https://user-images.githubusercontent.com/20795331/76954971-38653380-6909-11ea-9123-0051633ecbf8.png)

As this section is shared in the MF site, I don't think this is like anything in the MF style guide, in which case we probably shouldn't provide it as an example as not everyone is aware of the styleguide and they take our documentation as the guide on how things should look.

It's probably easier to just remove the document icon all together as it serves no real purpose, @donna-marijne any thoughts?

Reply:
A checkbox is an alternative that is suitable across both themes, however perhaps that only bloats the sample code unnecessarily?

Reply:
Just to add to this, I spoke to Roland and he had concerns regarding these icons as well.

Reply:
I'm fine to get rid of it, it adds nothing to the example after all.

Reply:
Removed here and in sample